homework.md

```diff
@@ -4,38 +4,38 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White I guess.
```

Comment:
why you guess? could be another colour(or colours), which one?

Reply:
Well. Purely technically speaking 'black' isn't a colour at all. :D So if in the context we refer to 'black' as not a colour then as an opposite any colour would do, even purple. :D

Reply:
I would say the question have a lack of context.
On this website they talk more about colour in a scientific and artistic perspective:
https://www.adobe.com/uk/creativecloud/design/discover/is-black-a-color.html

Reply:
I like your answers Jacint, brief & precise. Good job!

Reply:
Thank you Shadi!